### ISSUE TITLE
Category links / references infooter navigation underlined after update to 6.7

### ISSUE THREAD

**Issue by swagTKA** [shopware/platform#10683]
### Shopware Version

6.7.0.0

### Affected area / extension

Platform(Default)

### Actual behaviour

After the update to 6.7, the category links/references in the footer navigation are now suddenly permanently underlined.

This was not the case before or only if you hovered over the categories.

![Image](https://github.com/user-attachments/assets/c50ce0df-0757-4d27-9970-5937e2887ac3)

### Expected behaviour

Categories in the footer are not permanently underlined

### How to reproduce

1. Check Footer in older version
2. Update instance from 6.x to 6.7
3. Check Footer again

### Definition of Done

- [ ] An automated test covers the fix.
- [ ] Integration/E2E testing in staging is done.
- [ ] Changelog markdown file created or updated in /changelog/_unreleased directory.
- [ ] Developer documentation written/updated.
- [ ] End user and/or developer documentation written or updated with clarification.
- [ ] Regression test created to ensure bug does not reappear, and automated regression tests pass.
- [ ] Code changes resolve the reported bug without introducing new issues.

**Comment by ravenokko** [comment]
Closing this as it is an accessibility improvement to consistently, visually identify links. Using only color is a [failure](https://www.w3.org/WAI/WCAG22/Techniques/failures/F73.html) even combined with hover effects (which you cannot emulate if you're only using a keyboard).
